Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_Cloud_Cryptomining_-_Set_Verdict_README.md

```diff
@@ -1,9 +1,9 @@
 This playbook sets the alert's verdict as malicious if one of the following conditions is true:
-- If the source IP address is malicious
-- If the incident includes both "Unusual allocation of multiple cloud compute resources" AND "Cloud identity reached a throttling API rate" (medium/high severity)
-- If the incident includes both "Unusual allocation of multiple cloud compute resources" AND "Suspicious heavy allocation of compute resources - possible mining activity" 
-- If the incident includes "Unusual allocation of multiple cloud compute resources" with medium/high severity, the source ASN isn't known, and the source IP isn't known as well.
-- If the incident includes both "Unusual allocation of multiple cloud compute resources" AND "A cloud compute instance was created in a dormant region"
+1. If the source IP address is malicious
```

Comment:
This should not be a numbered list.  Numbered lists are only used when things have to be done in a certain order.  
This should be a bulleted list (as it was).